apps/client/src/pages/Dashboard/chartAndUpcoming/Chart/CustomTootip.tsx

```diff
@@ -0,0 +1,46 @@
+import { TooltipProps } from 'recharts'
+import PropTypes, { InferProps } from 'prop-types'
+import styles from './chat.module.scss'
+import Indicator from './images/Legend Indicator.svg?react'
+import Beak from './images/Bottom Beak.svg?react'
+import { Text } from 'components'
+
+interface DataPoint {
+  value?: number
+}
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+interface CustomTooltipProps extends TooltipProps<any, any> {
+  payload?: DataPoint[]
+}
+export const CustomTooltip: React.FC<CustomTooltipProps> = ({
+  active,
+  payload,
+}) => {
+  if (active && payload && payload.length) {
+    return (
+      <div className={styles.customTip}>
+        {/* <div className={styles.customTipDot}></div> */}
```

Comment:
needed?

Reply:
Yes it’s needed , is how the library accepts the customization

Reply:
I am referring to the comment